cmd/kubelet/app/server.go

```diff
@@ -966,9 +966,13 @@ func RunDockershim(c *componentconfig.KubeletConfiguration, dockershimRootDir st
 		SupportedPortForwardProtocols:   streaming.DefaultConfig.SupportedPortForwardProtocols,
 	}
 
+	cadvisorInterface, err := cadvisor.New(uint(c.CAdvisorPort), "docker", c.RootDirectory)
```

Comment:
@timstclair They are not running at same time. dockershim mode doesn't start kubelet components.